### DISCUSSION TITLE
Version 1.16.4

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#2890] [Changelog]
## Bugfixes
- #2888 Browser back buttons do not cause some addons to update. (prio-3)
- #2895 editor-theme3 has an incorrect note. (prio-4)

## Studio Migration
- #2886 dark-www does not make some parts of new studios dark. (prio-3)
- #2886 studio-tools buttons do not disappear after changing pages. (prio-2)

**Comment by Shluffy** [comment]
has this made the new record? `1.16.1`, `1.16.2`, `1.16.3`, and now `1.16.4`, compared to `1.14.1`, `1.14.2`, and `1.14.3`
